src/location-registry/models/location.py

```diff
@@ -1,29 +1,22 @@
 import pandas as pd
```

Comment:
- Great usage of the entity file for showcasing the attributes and functionalities of the location entity as per our class and Sequence Diagrams.
- Just curious, how does one make schema changes to the location entity. For example: if I want to remove the controversial _**mobility**_ field from the locations schema, how do I go about it?